glass/grf/_core.py

```diff
@@ -164,7 +164,7 @@ def compute(
 
     See Also
     --------
-    glass.grf.solve: Iterative solver for non-band-limited spectra.
+    :func:`glass.grf.solve`: Iterative solver for non-band-limited spectra.
```

Comment:
This is not what the numpydoc style guide says: https://numpydoc.readthedocs.io/en/latest/format.html#see-also

Reply:
Right, I guess because we are using napoleon in the end. I still think we should follow the style guide if we can, because at least there is a set of rules written down somewhere we can point to.